### DISCUSSION TITLE
[Docs] Add hint for removing "healthcheck.start_interval" for use with Podman and docker compose

### DISCUSSION THREAD

**Discussion by kastl-ars** [immich-app/immich#12638] [Feature Request]
### The bug

Using rootless Podman together with docker compose works like a charm. And it works with the Immich `docker-compose.yml`, but only once I comment out the `start_interval` for the postgres container.

It would be nice to add a small hint to the [documentation](https://immich.app/docs/install/docker-compose#step-3---start-the-containers) to comment out that line if an error like the following appears:

```
can't set healthcheck.start_interval as feature require Docker Engine v25 or later
```

### The OS that Immich Server is running on

openSUSE Tumbleweed

### Version of Immich Server

v1.115.0

### Version of Immich Mobile App

none

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Taken from https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml today, aka this should be from v1.115.0
```

### Your .env content

```Shell
Taken from https://github.com/immich-app/immich/releases/latest/download/example.env today, this should be from v1.115.0

Only change: I changed the IMMICH_VERSION to `v1.115.0`
```

### Reproduction steps

1. Run `docker compose up -d`
2. The error regarding the `start_interval` appears
3. Comment out the `start_interval` line
4. Run `docker compose up -d`
5. Party!
...

**Comment by mmomjian** [comment]
Feel free to add a FAQ entry about this.

**Reply by kastl-ars** [reply]
I added a hint to the docs:
https://github.com/immich-app/immich/pull/12643

(As there are other docker-related hints already)

**Comment by peterhirn** [comment]
https://github.com/containers/podman-compose/pull/780
